common/src/main/java/com/sequenceiq/cloudbreak/util/DatabaseUtil.java

```diff
@@ -24,4 +24,9 @@ public static void createSchemaIfNeeded(String dbType, String dbAddress, String
             }
         }
     }
+
+    public static String encapsulateContentForLikelinessQuery(String content) {
```

Comment:
I would move this to a separate class and use that as a component. it's easier to mock in tests. or you could convert this whole class to a component